migrations/1_initial_migration.js

```diff
@@ -1,6 +1,5 @@
-let Migrations = artifacts.require("Migrations");
+let Migrations = artifacts.require("./Migrations.sol");
```

Comment:
was the previous version of this supposed to work or was it a bug to just write "Migrations"?

Reply:
So, I still haven't found documentation of what causes the issue. A freshly init'd truffle package's boilerplate migration codes it as "./Migrations.sol"... and making the change to this format vs the old format resulted in our migrations acting as expected on the current truffle (old truffle worked with both).